native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -162,6 +166,24 @@ class BpkTextInput extends Component<Props, State> {
       valid,
     );
 
+    const renderExtraInfo = () => {
```

Comment:
I'm not sure that there is any benefit to this being a separate function. Maybe some readability gain from not having it all in-line, but it's not a massive component anyway

Reply:
Hmm yeah, I see your point, but to me the inline version is less readable:

```
        {valid === false &&
          validationMessage && (
            <BpkText textStyle="xs" style={styles.validationMessage}>
              {validationMessage}
            </BpkText>
          )}
        {valid !== false &&
          description && (
            <BpkText textStyle="xs" style={styles.description}>
              {description}
            </BpkText>
          )}
```

What do you think? cc @matteo-hertel @matthewdavidson @k0nserv

Reply:
I think it's fine as is 👍

Reply:
It makes the render method more readable, so gest a 👍 from me!